Auth0/Authentication.swift

```diff
@@ -155,7 +155,7 @@ public protocol Authentication: Trackable, Loggable {
         .start { print($0) }
      ```
 
-     you can also add additional attributes when creating the user
+     you can also add additional metadata when creating the user
```

Comment:
wouldn't "attributes" fit for both scenarios now?

Reply:
Sure, but the example code after this only changes metadata